### PR TITLE
[v6r11-onwards]CHANGE: Force use of TLSv1 from the client side

### PR DESCRIPTION
**Pull request by acasajus**
This just enables TLSv1 for outgoing connections via suds (mainly to connect to VOMS admin/whatever servers). DIRAC connections were already TLSv1

### CODE REVIEW

**Comment by atsareg** [comment]
Are you sure that changing the protocol to TLS is compatible with all the potential servers that we might need to work with ?

**Comment by acasajus** [comment]
Everyone should have migrated to TLS after the multiple hacks/bugs/flaws found on SSLv2/3.

File: Core/DISET/private/BaseClient.py [hunk 1]
```diff
@@ -98,6 +98,13 @@ def __discoverURL( self ):
       return S_ERROR( "URL is malformed: %s" % retVal[ 'Message' ] )
     self.__URLTuple = retVal[ 'Value' ]
     self._serviceName = self.__URLTuple[-1]
+    print self.__URLTuple[1:3]
```
Comment (chaen) [inline Core/DISET/private/BaseClient.py]: Do you really want a print ?

**Comment by acasajus** [comment]
Oops. Forgot to push last cleaning commit

**Comment by andresailer** [comment]
I pulled this branch; and at least this solves my problem with the desy voms-server
(cf. https://groups.google.com/forum/?hl=en#!topic/diracgrid-forum/VUGaSeIfXPU )

**Comment by acasajus** [comment]
You didn't enable any special configs. Just pulled the patch and connected with the VOMS server right? Can you ask the DESY admins what version of VOMSAdmin and JAVA are they using?

**Comment by andresailer** [comment]
Yes, just pulled, nothing added to config
VOMS Admin version 3.3.2 (https://grid-voms.desy.de:8443/voms/ilc/)
let me ask them about the java version...

**Comment by graciani** [comment]
To report here what was found out offline. Older Java versions are incompatible with recent OpenSSL version that implement stricter checks. This was causing TLSv1 handshakes with VOMS to fail in certain combinations.

**Pull request merged**